build.gradle

```diff
@@ -71,7 +71,7 @@ dependencies {
     annotationProcessor objectify
 
     compile         "com.google.appengine:appengine-api-1.0-sdk:${appengineVersion}",
-                    "com.google.appengine.tools:appengine-gcs-client:0.7",
+                    "com.google.appengine.tools:appengine-gcs-client:0.8",
```

Comment:
Is it necessary to update package in this PR, or should we move it to a separate PR?

Reply:
Somehow the dependency of the package is missing in the maven center: https://travis-ci.org/TEAMMATES/teammates/builds/644947137?utm_source=github_status&utm_medium=notification

The only why to make it build is to upgrade to a new version.

Reply:
OK, I think I will approve this PR after the travis issue being solved